android/app/build.gradle

```diff
@@ -41,6 +41,7 @@ android {
         applicationId "mk.webfactory.flutter_template"
         minSdkVersion 24
         targetSdkVersion 31
+        multiDexEnabled true
```

Comment:
It is enabled by default from a version I can't remember but lower than 24, did you put it because it was in the firebase installation steps or did you encounter an error without it?